HerPortal/Models/HomepageViewModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿using HerPortal.BusinessLogic.Models;
+
+namespace HerPortal.Models;
+
+public class HomepageViewModel
+{
+    public bool ShouldShowBanner { get; }
+
+    public HomepageViewModel(User user)
```

Comment:
This is fairly trivial at the moment, but it's still a good candidate for a unit test, to get that kicked off